packages/checkout-button-integration/jest.config.js

```diff
@@ -0,0 +1,17 @@
+module.exports = {
+    displayName: 'checkout-button-integration',
+    preset: '../../jest.preset.js',
+    globals: {
+        'ts-jest': {
+            tsconfig: '<rootDir>/tsconfig.spec.json',
+            diagnostics: false,
+        }
+    },
+    transform: {
```

Comment:
Do we need to setup these transforms here? We can just have them defined in preset file, unless they are specific per app?

Reply:
It's created by the generator. I'll remove 🙂